pkg/provider/bitbucketcloud/events.go

```diff
@@ -100,19 +116,32 @@ func (v *Provider) ParsePayload(ctx context.Context, run *params.Run, request *h
 		return &info.Event{}, err
 	}
 
-	allowed, err := v.checkFromPublicCloudIPS(ctx, run)
-	if err != nil {
-		return nil, err
-	}
-	if !allowed {
-		return nil, fmt.Errorf("payload is not coming from the public bitbucket cloud ips as defined here: %s",
-			bitbucketCloudIPrangesList)
-	}
+	// TODO: figure this out!
```

Comment:
Arf we reallly need to figure this out now before merging :D 

Without this anyone can use any openshift/kubernetes cluster to do anything they want.

Reply:
😄 ah ok ! let me look into it..

Reply:
updated! ptal :)